### DISCUSSION TITLE
how do i disable the background of the crosshair and i don't have the key "alt" on my mac

### DISCUSSION THREAD

**Discussion by sorrowist66** [lacymorrow/crossover#345] [Q&A] [answered]
?

**Comment by lacymorrow** [comment, marked as answer]
`Alt` on mac is labeled `Option`

To lock the crosshair in place and hide the background, press CTRL+Option+Shift+X

**Comment by SilvioLevy** [comment]
That does not work for me: pressing CTRL+Alt+Shift+X on Ubuntu linux does not make the background disappear. the window remains opaque. And the crosshair is only transparent with respect to the background. See the attachment please
![crossover_is_opaque](https://github.com/user-attachments/assets/83021cf2-7bdf-45f8-b2ab-f7847fc11a19)
